R/tm_g_scatterplotmatrix.R

```diff
@@ -147,38 +147,45 @@ ui_g_scatterplotmatrix <- function(id, ...) {
         )
       )
     ),
-    forms = teal::get_rcode_ui(ns("rcode")),
+    forms = teal.widgets::verbatim_popup_ui(ns("rcode"), "Show R code"),
     pre_output = args$pre_output,
     post_output = args$post_output
   )
 }
 
-srv_g_scatterplotmatrix <- function(id, datasets, reporter, variables, plot_height, plot_width) {
+srv_g_scatterplotmatrix <- function(id, data, reporter, filter_panel_api, variables, plot_height, plot_width) {
```

Comment:
From the scatterplotmatrix vignette - not happy about adding correlation

![image](https://user-images.githubusercontent.com/15201933/187468795-df07283d-48ed-4250-ad5c-8f015a2d5410.png)

Reply:
fixed, I do not like it is not working for factor variables with a few levels. correlations are assessed only for numeric variables, buuu.